content/docs/started/k8s/kfctl-istio-dex.md

```diff
@@ -37,6 +37,11 @@ Configuring your installation with {{% config-file-istio-dex %}} has a few optio
   1.3.1 with SDS enabled, which requires you to use Kubernetes 1.13 or later. You may need
   to add extra configurations to your Kubernetes as mentioned in
   [Istio's blog](https://istio.io/blog/2019/trustworthy-jwt-sds/).
+  The Istio community runs their test-infrastructure with the following API Server flags:
+  ```
+  "service-account-issuer": "kubernetes.default.svc"
```

Comment:
Can we clarify this further, by telling people which config file they need to update and where in the file they should put these flags? The easiest way to show the latter is to link to an example, so that we don't have to give the full hierarchy of names leading down to `apiServer`.